### DISCUSSION TITLE
style priority

### DISCUSSION THREAD

**Discussion by zeynab-rs** [quasarframework/quasar#11271] [General - Components / Directives / etc]
In Quasar 2.3.0, quasar styles prioritize relative to my styles in css folder. In Quasar v1, this was reverse. how to prioritize my styles?

**Comment by hawkeye64** [comment]
Add `!important` to them

**Reply by pdanpdan** [reply]
He's no talking about this - I think he's talking about the order the CSS files are loading.
@heyran we need details

**Reply by zeynab-rs** [reply]
*She :)
Exactly. Styles in css folder should overwrite Quasar styles as happened before in v1. But in v2, it is reversed.
I imported `quasar.variables.scss `and `app.scss` in `App.vue` file, and `app.scss` in `MainLayout.vue` in scoped mode to solve problem.
